app/utils/post/index.ts

```diff
@@ -152,7 +152,7 @@ export async function persistentNotificationsConfirmation(serverUrl: string, val
             });
             description = intl.formatMessage({
                 id: 'persistent_notifications.confirm.description',
-                defaultMessage: '@mentioned recipients will be notified every {interval, plural, one {minute} other {{interval} minutes}} until they’ve acknowledged or replied to the message.',
+                defaultMessage: '@username will be notified every {interval, plural, one {minute} other {{interval} minutes}} until they’ve acknowledged or replied to the message.',
```

Comment:
I think the idea was not so much `@username` literally, but `@username` as in, for example, `@john.doe` (the other user in the DM).

Reply:
@larkox so we wanna use the actual user's nam, right? @tanmaythole are you open to making this change?

Reply:
Hi @larkox ,
If there are multiple users mentioned, do we need to show only the first user's name or all users' names separated by commas?

Reply:
The logic should follow a similar flow and similar texts as what is done in this file in the server repo: `webapp/channels/src/components/persist_notification_confirm_modal.tsx`

I think that "unifying the logic" (do the same in mobile and web) would be best. Sorry for the back and forth!

Reply:
So this change is not required, right?

Reply:
From my perspective, a change is needed, but not exactly here. I would try to make the logic flow more similar to the web, so add some restructuring of the `if/else` here.

Also, just to align the texts, I would use the text in web, so... in this particular text, instead of `@mentioned` I would write it down as `Mentioned`.

Reply:
So, if you don't mind, can I work on that?

Reply:
Yes, please do. You can do it in the context of this PR :smile:

Reply:
Sure I will do that here